console/console-init/ui/yarn.lock

```diff
@@ -4580,6 +4580,13 @@ center-align@^0.1.1:
     align-text "^0.1.3"
```

Comment:
We don't need to commit this file.

Reply:
yes, we don't need this file change. @suyash-naithani please revert this file changes.

Reply:
Done